scout/server/blueprints/variants/utils.py

```diff
@@ -1,23 +1,20 @@
 # -*- coding: utf-8 -*-
+from scout.adapter import MongoAdapter
 
 
-def update_case_panels(store, case_obj):
+def update_case_panels(store: MongoAdapter, case_obj: dict):
     """Refresh case gene panels with info on if a panel was removed.
+    Also sets the key "latest_panels" for a case where the value is the latest version of the case panels
 
-    Also return these more populated panels for optional storage on the variant_obj.
```

Comment:
Mm, I have a feeling the intention was to let the caller decide wether to store or not, but this is very acceptable I think.